### PR TITLE
Update containerd installation instructions

### LINKED ISSUE #13049: Improve containerd instructions on k8s.io/docs/setup/cri/
The CRI installation page has installation instructions for containerd 1.1.2 using a tarball. Recently, containerd has been added to the docker-ce apt repository as `containerd.io` because docker-ce itself also uses containerd since version 18.09.

This means the installation instructions can be improved by installing containerd from the apt repository. 

**Page to Update:**
https://kubernetes.io/docs/setup/cri/

### PR DESCRIPTION
**Pull request by praseodym**
Use apt/yum instead of tarball-based installation.

Closes #13049.

### CODE REVIEW

**Comment by bradtopol** [comment]
Hi @mikebrow This PR is updating the install instrcutions for containerd.  Can you take a look and verify that the new instructions are correct.  When you are happy that they are correct please add a /lgtm and I'll take it from there.  Thanks!

File: content/en/docs/setup/cri.md [hunk 1]
```diff
@@ -223,36 +220,61 @@ EOF
...
+# Configure containerd
+mkdir -p /etc/containerd
+containerd config default > /etc/containerd/config.toml
+
+# Restart containerd
+systemctl restart containerd
+{{< /tab >}}
+{{< /tabs >}}
+
+To use the `systemd` cgroup driver, set `plugins.cri.systemd_cgroup = true` in `/etc/containerd/config.toml`.
```
Comment (mikebrow) [inline content/en/docs/setup/cri.md]: Suggest putting this... and the below "as well" instruction in a systemd section.

Comment (praseodym) [reply content/en/docs/setup/cri.md]: Done!

**Comment by bradtopol** [comment]
Thanks @mikebrow!
/lgtm
/approve

**Pull request merged**